src/EnergyPlus/ThermalComfort.cc

```diff
@@ -49,6 +49,7 @@
 #include <algorithm>
 #include <cassert>
 #include <cmath>
+#include <boost/math/tools/roots.hpp>
```

Comment:
Hmmm, I think this would be the first use of `boost` within the main EnergyPlus code. @Myoldmopar ok with that? I know it's used in Kiva, but I think that's the only place.